File: 119-pascals-triangle-ii/119-pascals-triangle-ii.cpp
class Solution {
public:
    vector<int> getRow(int rowIndex) {
        vector<int> dp{1};
        for (int i = 1; i <= rowIndex; i++) {
            vector<int> vect{1};
            for (int j = 1; j < i; j++) {
                vect.push_back(dp[j-1] + dp[j]);
            }
            vect.push_back(1);
            dp = vect;
        }
        return dp;
    }
};

File: 119-pascals-triangle-ii/119-pascals-triangle-ii.py
class Solution(object):
    def getRow(self, rowIndex):
        """
        :type rowIndex: int
        :rtype: List[int]
        """
        dp = [1]
        for i in range(rowIndex):
            temp = [1]
            for j in range(1, len(dp)):
                temp.append(dp[j-1] + dp[j])
            temp.append(1)
            dp = temp
        return dp